atomicapp/utils.py

```diff
@@ -114,6 +115,45 @@ def __init__(self, target_path, workdir=None):
             self.__workdir = workdir
 
     @staticmethod
+    def running_on_openshift():
+        """
+        If we can detect the openshift api endpoint from the
+        environment and connect to it then we are running in
+        openshift.
+        """
+
+        # Detect possible openshift api url, if none then not in openshift env
+        url = Utils.get_openshift_api_endpoint_from_env()
+        if not url:
+            return False
+
+        # Validate it as an openshift endpoint (could just be
+        # kubernetes). Don't worry about ssl verification for now
+        # as the openshift provider will do that.
+        try:
+            response = requests.get(url, verify=False)
+            if response.status_code == 200:
+                logger.debug("Detected environment: openshift pod")
+                return True
+        except:
+            return False
+        return False
+
+    @staticmethod
+    def get_openshift_api_endpoint_from_env():
```

Comment:
sorry about being nitpicky, but other function calls within Utils are camelCase.

@rtnpro i think you brought this up previously before, what does it say in pep-8 specifically? otherwise, we should change utils to conform to how rtnpro did his refactor and made everything snake_case